shapely/geometry/geo.py

```diff
@@ -32,7 +32,10 @@ def shape(context):
     elif geom_type == "linestring":
         return LineString(ob["coordinates"])
     elif geom_type == "polygon":
-        return Polygon(ob["coordinates"][0], ob["coordinates"][1:])
+        try:
+            return Polygon(ob["coordinates"][0], ob["coordinates"][1:])
+        except IndexError:
+            return Polygon()
```

Comment:
This will work, yes, but instead I think we should avoid the IndexError and do something like this instead

```
elif geom_type == "polygon":
    if not ob["coordinates"]:
        return Polygon()
    else:
        ...
```

Reply:
the problem is that `ob["coordinates"]` exist.
in fact the dict is like `{"coordinates":()}`. But we could like 
```python
elif geom_type == "polygon":
    if len(ob["coordinates"])==0:
        return Polygon()
```

But then i personally find it simpler, and a bit more pythonic but that's totally subjective, to have a try except statement.

Reply:
fixed